integration/ui/src/main/resources/PhenoTips/VariantDisplay.xml

```diff
@@ -462,8 +462,8 @@
       $xwiki.ssx.use('PhenoTips.VariantDisplay')##
       $xwiki.jsx.use('PhenoTips.VariantTableWidget', {'minify' : false})##
       $xwiki.ssx.use('PhenoTips.VariantTableWidget')##
-      === Top genes in VCF ===
-      {{html clean=false}}&lt;p class=box hint"&gt;Gene scores are computed from the uploaded VCF file and patient phenotypes using &lt;a href="http://www.sanger.ac.uk/resources/software/exomiser/" target="_blank"&gt;the Exomiser&lt;/a&gt;. Each score reflects the phenotypic relevance of the gene and the harmfulness and allele frequency of the variants.&lt;/p&gt;
+      === $services.localization.render('phenotips.variantDisplay.topGenes.header') ===
```

Comment:
Don't do things like this, a unitary message shouldn't be split into multiple translations. There are several issues here:
- you forgot to translate "the Exomiser" text inside the link
- without knowing how the three pieces fit together, a translator wouldn't know how to translate a text ending with "... using the"
- not all languages use the same topic, so for example in German this would sound like "werden... von Exomiser berechnet.", and there's no way to put "berechnet" in the right place with these keys
- not all languages use the same punctuation marks, so it's not guaranteed that a dot in that place is correct; In Japanese, they use `。`
- the key should end with `.hint`, since that's a sub-part of that element; see the [localization conventions](http://dev.xwiki.org/xwiki/bin/Community/L10N+Conventions#HRulesforstandardformelements)

Do this:

    $services.localization.render('phenotips.variants.topGenes.exomiser.hint', ['<a href="http://www.sanger.ac.uk/resources/software/exomiser/" target="_blank">', '</a>'])

    phenotips.variants.topGenes.exomiser.hint=Gene scores are computed from the uploaded VCF file and patient phenotypes using {0}the Exomiser{1}. Each score reflects the phenotypic relevance of the gene and the harmfulness and allele frequency of the variants.